yb-voyager/cmd/assessMigrationCommandBulk.go

```diff
@@ -200,10 +201,10 @@ func buildCommandArguments(dbConfig AssessMigrationDBConfig, exportDirPath strin
 		args = append(args, "--source-db-port", dbConfig.Port)
 	}
 
-	// Always safe to use --start-clean and --yes to cleanup if there is some state from previous runs in export-dir
+	// Always safe to use --start-clean to cleanup if there is some state from previous runs in export-dir
 	// since bulk command has separate check to decide beforehand whether the report exists or assessment needs to be performed.
-	args = append(args, "--start-clean", "true",
-		"--yes", "true")
+	args = append(args, "--start-clean", "true")
+	args = append(args, "--yes", lo.Ternary(utils.DoNotPrompt, "true", "false"))
```

Comment:
Does `--yes false/true` work? Pls verify this. 
nit: an easier, more readable way is just 
```
if utils.DoNotPrompt: 
    args = append(args, "--yes")
```

Reply:
oh you are right, that does work. thanks
I supposed this flag to be BoolString as we have for `--start-clean true`...